rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -945,23 +962,24 @@ bool ppu_interpreter_fast::VMADDFP(ppu_thread& ppu, ppu_opcode_t op)
 	const auto b = ppu.vr[op.vc].vf;
 	const auto c = ppu.vr[op.vb].vf;
 	const auto result = _mm_add_ps(_mm_mul_ps(a, b), c);
-	ppu.vr[op.vd] = vec_handle_nan(result, a, b, c);
+	ppu.vr[op.vd] = vec_handle_nan(result);
```

Comment:
why remove args here?

Reply:
This is a fast interpreter's exclusive path, adding them would make it slower. I prefer it to be precise path currently, although handlers which are shared across precise and fast interpreters use the accurate NaN handling.

Reply:
ahh cool